habitat/configs/tasks/rearrange/rearrange_easy.yaml

```diff
@@ -0,0 +1,136 @@
+GYM:
```

Comment:
@vincentpierre Should this file be created here?
There is a similar file `habitat/config/tasks/rearrange_easy.yaml`.
Note, the only difference between this file and file `habitat/config/tasks/rearrange_easy.yaml` is TASK_SPEC_BASE_PATH key.

Reply:
Thank you for spotting that. There must have been a merge issue on my end, I will investigate.